winpr/libwinpr/sspi/NTLM/ntlm_compute.c

```diff
@@ -454,21 +453,21 @@ int ntlm_compute_ntlm_v2_response(NTLM_CONTEXT* context)
 	CopyMemory(blob, context->ServerChallenge, 8);
 	CopyMemory(&blob[8], ntlm_v2_temp.pvBuffer, ntlm_v2_temp.cbBuffer);
 	winpr_HMAC(WINPR_MD_MD5, (BYTE*)context->NtlmV2Hash, WINPR_MD5_DIGEST_LENGTH,
-	           (BYTE*)ntlm_v2_temp_chal.pvBuffer, ntlm_v2_temp_chal.cbBuffer, (BYTE*)nt_proof_str,
-	           WINPR_MD5_DIGEST_LENGTH);
+	           (BYTE*)ntlm_v2_temp_chal.pvBuffer, ntlm_v2_temp_chal.cbBuffer,
+	           (BYTE*)context->NtProofString, WINPR_MD5_DIGEST_LENGTH);
```

Comment:
unnecessary cast

Reply:
length is inconsistent, here `WINPR_MD5_DIGEST_LENGTH` is uses, below a magic number, ...